element_electrode_localization/electrode_localization.py

```diff
@@ -9,7 +9,7 @@
 
 from . import coordinate_framework
 
-schema = dj.schema()
+schema = dj.Schema()
 
 log = logging.getLogger(__name__)
```

Comment:
```suggestion
log = dj.logger
```
We can update this here while we're updating the Element